templates/emails/collective.virtualcard.deleted.hbs

```diff
@@ -14,4 +14,8 @@ Subject: ⚠ {{{collective.name}}}'s {{virtualCard.last4}} card was deleted
 <blockquote style="color: #6a737d;font-size: 16px;text-align: left;padding: 0.5em 0.75em;margin: 0 24px;border-left: 3px solid #e4e4e4;white-space: pre-line;font-style: italic;">{{message}}</blockquote>
 {{/if}}
 
+{{#if hostCollective}}
+  <p>Host Collective: {{> linkCollective collective=hostCollective}}</p>
```

Comment:
Could we either clear the style property of the other P tag or add the same style here to make it consistent?

Reply:
Done. It looks better now with; 

![image](https://github.com/opencollective/opencollective-api/assets/12435965/b0f1ec50-efb9-4d09-a63d-212e0a1faadf)